dot/rpc/modules/system_integration_test.go

```diff
@@ -436,7 +436,8 @@ func TestSyncState(t *testing.T) {
 }
 
 func TestLocalListenAddresses(t *testing.T) {
-	ma, err := multiaddr.NewMultiaddr("/ip4/127.0.0.1/tcp/7001/p2p/12D3KooWCYyh5xoAc5oRyiGU4d9ktcqFQ23JjitNFR6bEcbw7YdN")
+	ma, err := multiaddr.NewMultiaddr("/ip4/127.0.0." +
+		"1/tcp/7001/p2p/12D3KooWCYyh5xoAc5oRyiGU4d9ktcqFQ23JjitNFR6bEcbw7YdN")
```

Comment:
nit slightly better split

```suggestion
	ma, err := multiaddr.NewMultiaddr("/ip4/127.0.0.1/tcp/7001/p2p/"+
		"12D3KooWCYyh5xoAc5oRyiGU4d9ktcqFQ23JjitNFR6bEcbw7YdN")
```